open_humans/templates/email/welcome.txt

```diff
@@ -0,0 +1,34 @@
+Thank you for joining the Open Humans community!
+
+To make sure you get future emails from us, add our email to your “contacts”.
+
+{% if newsletter %}You’ve signed up for news and updates, but without this
+step our emails might be “filtered” by your email client.{% else %}
+Without this step our emails might be “filtered” by your email client. You’ll
+also need to sign up for “news and updates” on your account settings. We really
+don’t email much! We email to tell Members about new studies and activities,
+and give updates about progress in current studies.{% endif %}
+
+We believe research should go both ways: find studies to join and share data
+with, and studies that will share data with you. Build a profile that connects
+data across studies!
+
+You can even choose to make some data sets public. Joining our Public Sharing
+Data Study turns on the “public sharing” feature. You’ll be able to turn
+“public” status on or off for each of your data sets.
+
+Get started by visiting our Welcome page[1].
```

Comment:
Hm... limitations of text. I think it's better to do: `Get started by visiting our Welcome page: {{ welcome_page_url }}`

Also, above:
`Joining our Public Data Sharing study turns on the "public sharing" feature: {{ public_data_url }} You'll be able to turn "public" status on or off for each of your data sets.`